test/e2e/scalability/density.go

```diff
@@ -650,6 +684,8 @@ var _ = SIGDescribe("Density", func() {
 				}
 
 				// Create some additional pods with throughput ~5 pods/sec.
+				latencyPodStartupPhase := testPhaseDurations.StartPhase(800, "starting latency pods")
```

Comment:
'Latency pods creation'? To follow the pattern elsewhere.

Reply:
Done.